pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -53,11 +53,12 @@ var (
 
 // GetPolicyLabels returns a LabelArray for the given namespace and name.
 func GetPolicyLabels(ns, name string, uid types.UID, derivedFrom string) labels.LabelArray {
```

Comment:
please add a unit test for this function to make sure the order forever is correct,

Reply:
There are many already, adjusted them accordingly.